types/profiles/v1/profiles.go

```diff
@@ -87,7 +89,7 @@ const (
 // USAGE:
 //  Already mentioned in PlacementPolicyKey
 type PlacementPolicyValue string
-
+//SpreadReplicaPPV holds placement policy value for spread-replica.
 const (
 	SpreadReplicaPPV PlacementPolicyValue = "spread-replica"
 	StickyReplicaPPV PlacementPolicyValue = "sticky-replica"
```

Comment:
Possible linting issue with this constant. Please write comments for each constant